.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
```

Comment:
let's remove this line, I think it's a lot easier for us to determine this than for a user.